android/modules/platform/src/java/ti/modules/titanium/platform/DisplayCapsProxy.java

```diff
@@ -67,10 +67,20 @@ public String getDensity() {
 				return "high";
 			case DisplayMetrics.DENSITY_MEDIUM :
 				return "medium";
-			case 320 : // DisplayMetrics.DENSITY_XHIGH (API 9)
+			case 213: //TV
 				return "xhigh";
+			case 280: //Introduce in API 22.
+				return "xhigh";
+			case 320:
+				return "xhigh";
+			case 400:
```

Comment:
Shouldn't this return '400' to be in line with http://developer.android.com/reference/android/util/DisplayMetrics.html?